sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/TaskResourceMapper.java

```diff
@@ -230,9 +230,11 @@ private static OfferEvaluationStage toEvaluationStage(
             Optional<String> resourceId,
             Optional<String> persistenceId) {
         if (resourceSpec instanceof NamedVIPSpec) {
-            return new NamedVIPEvaluationStage((NamedVIPSpec) resourceSpec, taskSpecName, resourceId);
+            NamedVIPSpec namedVIPSpec = (NamedVIPSpec) resourceSpec;
```

Comment:
This was done during the "major" refactor that removed `DefaultOfferRequirementProvider` and the like. I can reinstate these methods into the `ResouceSpec` classes if y'all want.